cfgov/cfgov/urls.py

```diff
@@ -412,7 +429,8 @@
         url(r'^(?i)obtener-respuestas/api/autocomplete-(?P<language>es)/$',
             ask_autocomplete, name='ask-autocomplete-es'),
     ]
-    urlpatterns += ask_patterns
+
+urlpatterns += ask_patterns
```

Comment:
so ask_patterns gets defined, but the urls remain empty (or inactive?) until the flag kicks in?